### ISSUE TITLE
Broken tests: CheckStartupEventLogMessage et al

### ISSUE THREAD

**Issue by ryanbrandenburg** [aspnet/IISIntegration#1073]
This test [fails](http://aspnetci/viewLog.html?buildId=507770&buildTypeId=XPlat_HighSierraOsxUniverse) occasionally with the following error:
```
System.AggregateException : One or more errors occurred. (COM is not supported) (COM is not supported)
---- System.PlatformNotSupportedException : COM is not supported
---- System.PlatformNotSupportedException : COM is not supported
----- Inner Stack Trace #1 (System.PlatformNotSupportedException) -----
----- Inner Stack Trace #2 (System.PlatformNotSupportedException) -----
------- Stdout: -------
| [0.004s] TestLifetime Information: Starting test CheckStartupEventLogMessage at 2018-07-17T14:42:33
| [0.009s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Information: Deploying [Variation] :: ServerType=IIS, Runtime=CoreClr, Arch=x64, BaseUrlHint=, Publish=True
| [0.010s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Information: Removing environment variable ASPNETCORE_ENVIRONMENT
| [0.010s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Information: Executing command dotnet publish  --output "/Users/aspnetci/Documents/buildAgent/temp/buildTmp/285daa78-28d6-4dfe-80a1-0c3e1cbc0162" --framework netcoreapp2.2 --configuration Release --no-restore -p:VerifyMatchingImplicitPackageVersion=false 
| [0.488s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Information: dotnet-publish stdout: Microsoft (R) Build Engine version 15.7.179.6572 for .NET Core
| [0.488s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Information: dotnet-publish stdout: Copyright (C) Microsoft Corporation. All rights reserved.
| [4.302s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Information: dotnet-publish stdout:   TestTasks -> /Users/aspnetci/Documents/buildAgent/work/33bdfc1cae7b2a38/modules/IISIntegration/test/TestTasks/bin/Release/netcoreapp2.2/TestTasks.dll
| [4.542s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Information: dotnet-publish stdout:   Microsoft.AspNetCore.Server.IIS -> /Users/aspnetci/Documents/buildAgent/work/33bdfc1cae7b2a38/modules/IISIntegration/src/Microsoft.AspNetCore.Server.IIS/bin/Release/netstandard2.0/Microsoft.AspNetCore.Server.IIS.dll
| [5.341s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Information: dotnet-publish stdout:   InProcessWebSite -> /Users/aspnetci/Documents/buildAgent/work/33bdfc1cae7b2a38/modules/IISIntegration/test/WebSites/InProcessWebSite/bin/Release/netcoreapp2.2/InProcessWebSite.dll
| [6.676s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Information: dotnet-publish stdout:   InProcessWebSite -> /Users/aspnetci/Documents/buildAgent/temp/buildTmp/285daa78-28d6-4dfe-80a1-0c3e1cbc0162/
| [6.752s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Information: dotnet publish finished with exit code : 0
| [6.753s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to backup apphost.config: Could not find file '/System/inetsrv/config/applicationhost.config'.
| [7.003s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to backup apphost.config: Could not find file '/System/inetsrv/config/applicationhost.config'.
| [7.253s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to backup apphost.config: Could not find file '/System/inetsrv/config/applicationhost.config'.
| [7.503s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to backup apphost.config: Could not find file '/System/inetsrv/config/applicationhost.config'.
| [7.754s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to backup apphost.config: Could not find file '/System/inetsrv/config/applicationhost.config'.
| [8.004s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to backup apphost.config: Could not find file '/System/inetsrv/config/applicationhost.config'.
| [8.255s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to backup apphost.config: Could not find file '/System/inetsrv/config/applicationhost.config'.
| [8.505s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to backup apphost.config: Could not find file '/System/inetsrv/config/applicationhost.config'.
| [8.755s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to backup apphost.config: Could not find file '/System/inetsrv/config/applicationhost.config'.
| [9.006s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to backup apphost.config: Could not find file '/System/inetsrv/config/applicationhost.config'.
| [9.256s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Information: Backed up /System/inetsrv/config/applicationhost.config to /System/inetsrv/config/applicationhost.config.HttpTestSite20180717074233backup
| [9.256s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to copy IIS.config to apphost.config: Could not find a part of the path '/System/inetsrv/config/applicationhost.config'.
| [9.506s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to copy IIS.config to apphost.config: Could not find a part of the path '/System/inetsrv/config/applicationhost.config'.
| [9.757s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to copy IIS.config to apphost.config: Could not find a part of the path '/System/inetsrv/config/applicationhost.config'.
| [10.007s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to copy IIS.config to apphost.config: Could not find a part of the path '/System/inetsrv/config/applicationhost.config'.
| [10.257s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to copy IIS.config to apphost.config: Could not find a part of the path '/System/inetsrv/config/applicationhost.config'.
| [10.508s] Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer Warning: Failed to copy IIS.con
...
```

CC @Eilon

**Comment by ryanbrandenburg** [comment]
CC @Eilon to triage this.

**Comment by jkotalik** [comment]
Will be resolved by #1056 as this test shouldn't be running on OSX

**Comment by jkotalik** [comment]
Tests on mac have been skipped. If other test fails, most likely it is for a different issue.
